src/client/eppo-client.ts

```diff
@@ -501,12 +501,31 @@ export default class EppoClient {
     return { variation, action };
   }
 
+  getBestBanditAction(
```

Comment:
What do you think about adding a parameter to `getBanditAction` instead of creating a new function?

Maybe use an enum:
```ts
enum BanditSelectionMethod {
  WeightedProbabilistic,
  Best,
}
```

Also would be great to document the difference. Otherwise, presented with two functions `getBanditAction` and `getBestBanditAction`, I'm pretty sure which one users are picking 😅 and it's not the best choice 😝

Reply:
This was my first idea as well, and by overriding gamma to be `Infinity` nothing would need to be changed. However, then we'd be doing a lot of things we don't want to (or using flow control to skip them) including ~variation assignment~, logging checks, scoring and shuffling, bumping up tiebreaking, etc. 

After seeing it in action, I think the separate way with the reusable methods is the way to go and still clean.